docs/migration-guide-2.3-2.4.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Migration Guide 2.3-2.4
+---
+
+In Kyma 2.0, we introduced a [new way to reach registered services and some Application Connectivity improvements](https://kyma-project.io/blog/2021/12/7/release-notes-20/#application-connectivity), which were related to [Service Catalog removal](https://kyma-project.io/blog/2021/12/7/release-notes-20/#service-catalog-deprecation-update).
+We then also informed you we'd be removing the support for the old way soon and encouraged you to switch to the new flow making use of Central Application Gateway. In this release, we're following up on this promise and switching off the support for the old flow. 
+
+Due to the removal of the deprecated components (such as Application Operator, Application Broker, Rafter, and Service Catalog), certain resources which were previously created in your cluster are now obsolete and must be deleted if you made use of the old flow using these components.
+When you upgrade from Kyma 2.3 to 2.4, either run the script [`2.3-2.4-cleanup-orphaned-svcat-resources.sh`](assets/2.3-2.4-cleanup-orphaned-svcat-resources.sh) or perform the required steps from that script manually.
+
```

Comment:
"Due to the removal of the deprecated components (...) must be deleted (...) run the script (...)
TBH I read this as an obligation.

Users should conciously clean it up if they dont use those components anymore!
I.e running the script will break you application if you use service instances in you workloads ( for example functions )

Reply:
@PK85 @klaudiagrz

Reply:
I can add this, but they should already be aware of this since we announced that the old flow was deprecated and we mention at the beginning of the guide that we now removed support for it. 

And good point with the obligation. After Jan's comment, I changed the word from "must" to "should".

Reply:
Added a caution note at the end.